kitchen.dokken.yml

```diff
@@ -4,6 +4,7 @@ driver:
   privileged: true  # because Docker and SystemD
   chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
   chef_license: accept-no-persist
+  install_strategy: always
```

Comment:
Any particular reason why this is set?

Reply:
I was getting invalid chef cert.pem files.  I thought this was doing a reinstall.  It does fix the error but was not intended as a permanent thing.

Reply:
I'd recommend it be removed before this gets merged.

Reply:
Removed it.